actors/miner/src/emit.rs

```diff
@@ -0,0 +1,27 @@
+// A namespace for helpers that build and emit Miner Actor events.
+
+use fil_actors_runtime::runtime::Runtime;
+use fil_actors_runtime::{ActorError, EventBuilder};
+use fvm_shared::sector::SectorID;
+
+/// Indicates a sector has been pre-committed.
+pub fn sector_precommited(rt: &impl Runtime, id: SectorID) -> Result<(), ActorError> {
+    rt.emit_event(
+        &EventBuilder::new()
+            .typ("sector-precommited")
+            .field_indexed("miner", &id.miner)
+            .field_indexed("sector", &id.number)
+            .build()?,
+    )
+}
+
+/// Indicates a sector has been proven.
+pub fn sector_proven(rt: &impl Runtime, id: SectorID) -> Result<(), ActorError> {
```

Comment:
Please rename to sector-activated. Today these always happen at the same time, but in the future we may separate them. If we do, it's likely that most users of the event really care about the activation, rather than an un-activated proof. We'll reserve sector-proven for that case.

Reply:
Done.